src/azul/indexer/__init__.py

```diff
@@ -403,6 +406,27 @@ def drs_path(self, manifest_entry: JSON) -> Optional[str]:
         """
         raise NotImplementedError
 
+    def _reject_joiner(self, haystack: CompositeJSON):
+        if isinstance(haystack, dict):
+            # Since the keys in the metadata files and manifest are pre-defined,
+            # we save some time here by looking for the joiner in the keys.
+            for v in haystack.values():
+                self._reject_joiner(v)
+        elif isinstance(haystack, list):
+            for v in haystack:
+                self._reject_joiner(v)
+        elif isinstance(haystack, str):
+            if config.manifest_value_joiner in haystack:
```

Comment:
We want to suppress `||` (string of length two) in metadata. We want to use ` || ` (a string of length two) between values in a column.